### ISSUE TITLE
P.87 to P.88 the example "does not delete the project" should be under the context "as a guest"

### ISSUE THREAD

**Issue by dannysiu392005** [everydayrails/everydayrails-rspec-2017#9]
So, the following should be amended from
```ruby
  context "as a guest" do
    before do
      @project = FactoryGirl.create(:project)
    end

    it "returns a 302 response" do
      delete :destroy, params: { id: @project.id }
      expect(response).to have_http_status "302"
    end

    it "redirects to the sign-in page" do
      delete :destroy, params: { id: @project.id }
      expect(response).to redirect_to "/users/sign_in"
    end
  end
  
  it "does not delete the project" do
    expect {
      delete :destroy, params: { id: @project.id }
    }.to_not change(Project, :count)
  end
```

to

```ruby
  context "as a guest" do
    before do
      @project = FactoryGirl.create(:project)
    end

    it "returns a 302 response" do
      delete :destroy, params: { id: @project.id }
      expect(response).to have_http_status "302"
    end

    it "redirects to the sign-in page" do
      delete :destroy, params: { id: @project.id }
      expect(response).to redirect_to "/users/sign_in"
    end

    it "does not delete the project" do
      expect {
        delete :destroy, params: { id: @project.id }
      }.to_not change(Project, :count)
    end
  end
```

**Comment by ruralocity** [comment]
This will be fixed in the next release, as noted in #8. Thanks.

**Comment by ruralocity** [comment]
Fixed in today's release (July 11, 2017).
